src/plugins/navigation/public/top_nav_menu/top_nav_menu_badges.tsx

```diff
@@ -22,7 +22,7 @@ export const TopNavMenuBadges = ({ badges }: { badges: TopNavMenuBadgeProps[] |
   );
 };
 
-function createBadge(
+export function createBadge(
```

Comment:
Looks like you decided to not call `createBadge` anymore? So we shouldn't export it